### PR TITLE
Added additional methods to the Request class #14

### PR DESCRIPTION
**Pull request by TahirGuliyev**
This commit adds the following methods to the Request class:

- get: This method attempts to fetch a value from the $_GET superglobal using the provided key. If the key does not exist, it returns a provided default value. The returned value is sanitized for special characters.

- post: Similar to the 'get' method, this method tries to retrieve a value from the $_POST superglobal using the given key. If the key doesn't exist, it returns a provided default value. The returned value is also sanitized for special characters.

- cookie: This method fetches a value from the $_COOKIE superglobal using a specified key. If the key does not exist, it returns a provided default value. The returned value is sanitized for special characters.

- has: This method checks if a specific key or array of keys exists in the $_GET or $_POST superglobals. It utilizes the 'get' and 'post' methods for this check and returns true if the key(s) exist, otherwise false.

### CODE REVIEW

**Comment by devalirzayev** [comment]
@TahirGuliyev eladir 👍🏻 

Bir dene menim sehvim olub. Eslinde ne get, ne de post data sanitize olunmali deyil. Nece varsa ele de gelmelidiler. Amma set olunmayibsa ve ya boshdursa ( example, !isset($_GET[$key]) || trim( $_GET[$key] ) === "" ) o zaman $default qaytaraq. Bunu bir dene deyishersen kodunda zehmet olmasa.

File: app/Core/Request.php [hunk 1]
```diff
@@ -60,4 +60,38 @@ public static function rawBody(): string
...
+        }
+
+        foreach ($keys as $key) {
+            if ($type == 'GET' && self::get($key) === null) {
+                return false;
+            }
+
+            if ($type == 'POST' && self::post($key) === null) {
+                return false;
+            }
```
Comment (devalirzayev) [inline app/Core/Request.php:92]: burani bele yazsan ishleyer mence:

```php
if ( self::$type($key) === null ) {
    return false;
}
```

**Review by devalirzayev — CHANGES_REQUESTED** [review]

**Comment by TahirGuliyev** [comment]
Çaşıb bağladım pull requesti :) Yenidən açıram.

File: app/Core/Request.php (hunk 1 above)
Comment (TahirGuliyev) [reply app/Core/Request.php]: Hazırdır!

**Review by devalirzayev — APPROVED** [review]

**Pull request merged**